lib/index.js

```diff
@@ -49,6 +51,7 @@ exports.register = (server, options, next) => {
       headers['X-Rate-Limit-Remaining'] = rate.remaining;
       headers['X-Rate-Limit-Limit'] = rate.limit;
       headers['X-Rate-Limit-Reset'] = rate.reset;
+      headers['X-Rate-Limit-Window'] = rate.window;
```

Comment:
The primary reason was that this would be the easiest way to pass down this value (there's another way too). 

But there's also no harm in providing this information, so I thought why not just provide it (better than realizing later that we should have included this). 

What are your thoughts?